image/image.go

```diff
@@ -263,3 +263,13 @@ func NewImgJSON(src []byte) (*Image, error) {
 	}
 	return ret, nil
 }
+
+// Check wheather id is a valid image ID or not
+func ValidateID(id string) error {
```

Comment:
yeah, I just thought I'd mention it. I was thinking of creating an (non related) issue because I noticed some parts of the code used `stringid.TruncateID()` and some parts used a literal `id[:12]`, then I stumbled upon this PR :smile: